### ISSUE TITLE
Open sourcing of TraceTogether

### ISSUE THREAD

**Issue by waqarrashid33** [baradhili/TogetherTrace#1]
https://bluetrace.io/

**Comment by baradhili** [comment]
but they are sitting on the code, which is really irritating...... 
also their model has tight control by the government.. which is fine for Singapore - not so good for countries where there is movement between states.. or even movement between countries

**Comment by waqarrashid33** [comment]
How about reversing it? I have access to the APK file

**Comment by waqarrashid33** [comment]
Also I think lots of their code would be in the back end, not sure if they will ever release that.

**Comment by baradhili** [comment]
might be worth it.. I'm thinking the backend might not have much in it.. I think the reason they haven't open-sourced is because their keys are embedded in the code..

I'm going to spend time today thinking out what the protocol might need - but I've dropped them a line, and I'm in the same timezone so might actually get a response..

**Comment by waqarrashid33** [comment]
Lots of other people talking about, might be a better idea to join a pre existing team

https://github.com/search?q=tracetogether&type=Issues

**Comment by baradhili** [comment]
I'm thinking https://github.com/memiah/contact-tracer might be the furthest along

**Comment by waqarrashid33** [comment]
Yeah. I already forked it. But I am not an android developer. I will have a look at the stuff that needs to be done and see if I can do any of them.
